src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ImportServiceTests.cs

```diff
@@ -0,0 +1,77 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Admin.Controllers.Api.BAU;
+using GovUk.Education.ExploreEducationStatistics.Admin.Models;
+using GovUk.Education.ExploreEducationStatistics.Admin.Security;
+using GovUk.Education.ExploreEducationStatistics.Admin.Services;
+using GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Common.Services.Interfaces.Security;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Moq;
+using Xunit;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Tests.Services
+{
+    public class ImportServiceTests
+    {
+        [Fact]
+        public async void CancelFileImport()
+        {
+            var userService = new Mock<IUserService>();
+
+            var cancelRequest = new ReleaseFileImportInfo
+            {
+                ReleaseId = Guid.NewGuid(),
+                DataFileName = "my_data_file.csv"
+            };
+
+            userService
+                .Setup(s => s.MatchesPolicy(cancelRequest, SecurityPolicies.CanCancelOngoingImports))
+                .ReturnsAsync(true);
+
+            var service = new ImportService(
```

Comment:
Would refactor the construction into a `BuildImportService` method so that we don't have to manually (and potentially error prone) passi in of null values. 

I think in most tests we've also been fully setting up default mocks for all of the parameters. In quite a few tests, we've also been using named parameters at the call sites to make it pretty easy to override those defaults. See `ReleaseServiceTests.BuildReleaseService` for an example

Reply:
LOVE IT!!!  Done.